src/components/Form.js

```diff
@@ -20,12 +20,18 @@ function Form({
   list,
   loading,
   onFetchSearchList,
+  activeSearchListElement,
+  onSetActiveSearchListElement,
 }) {
+  const [selectedCity, setSelectedCity] = useState(null);
+
   const handleSetQuery = (e) => {
     if (!/^[a-zA-Z\s]*$/g.test(e.target.value)) return;
     setSearchedQuery(e.target.value);
 
     onFetchSearchList(e.target.value);
+    onSetActiveSearchListElement(-activeSearchListElement);
```

Comment:
Why do we use `-` sign here in front of `activeSearchListElement`?

Reply:
It is used in order to set activeSearchListElement to 0 after each character input, so that user can always start navigating from the beginning of the list.